### DISCUSSION TITLE
[NEW RELEASE] Introducing MARS5, open-source, insanely prosodic text-to-speech (TTS) model.

### DISCUSSION THREAD

**Discussion by akshhack** [rhasspy/piper#522] [Show and tell]
![mars-banner](https://github.com/rhasspy/piper/assets/21692676/c29d90eb-43e5-4411-8a9c-339f4b45bb54)

Hey community members! 👋 

We, at [CAMB.AI](https://www.camb.ai), are super stoked to announce the open source release of **MARS5**, a new speech emulation model that is able to replicate even extremely tough prosody like sports commentary, anime, movies with just a few seconds of audio reference. 

_Check out our release:_ https://github.com/Camb-ai/MARS5-TTS

Watch our demo here:

https://github.com/rhasspy/piper/assets/21692676/0c747da1-5216-48ce-8dfc-f08cd00756ae

and the full release video: https://www.youtube.com/watch?v=bmJSLPYrKtE

We're excited to hear feedback and see the community build on top of it!

Quick links:
**Discord:** https://discord.gg/ZzsKTAKM
**Github:** https://www.github.com/camb-ai/mars5-tts
**Website:** https://www.camb.ai/
**Youtube:** https://www.youtube.com/@camb-ai

**Comment by luiscarlos2000** [comment]
Looks interesting as it may seem. Can we add this training model to Piper besides VITS and the proposed VITS2? It could be interesting how it works.
